app/components/CardBet.tsx

```diff
@@ -141,7 +142,12 @@ export const CardBet = ({ userPosition }: BetProps) => {
             </p>
           </div>
           <div className="flex space-x-4">
-            <div className="size-[40px] rounded-8 bg-text-low-em bg-gradient-to-r from-[#cb8fc1] to-[#b459c6]" />
+            <Thumbnail
+              width={40}
+              height={40}
+              className="size-[40px] rounded-8"
```

Comment:
Minor, non-blocking

Maybe we can abstract the `40` into a const and reuse, something like `THUMBNAIL_SIZE`

Reply:
Will leave it as it is for now